packages/dds/sequence/src/index.ts

```diff
@@ -81,6 +81,7 @@ export {
 	getTextAndMarkers,
 	ISharedString,
 	SharedString,
+	SharedString as SharedStringClass,
```

Comment:
This looks like a breaking change in a patch. :/

Reply:
My intent was to re-export the same class with a different name. Is that breaking? I guess the other question is if it's actually safe. :)

Reply:
That is breaking. Anyone using the old name won't find it. Or if you export something different, then they are getting something different.